### PR TITLE
Move helper methods into services

### PR DESCRIPTION
**Pull request by Fryguy**
@jrafanie Please review.

### CODE REVIEW

File: lib/multi_repo/service/github.rb [hunk 1]
```diff
@@ -138,6 +138,44 @@ def delete_label!(repo_name, label)
...
+        client.add_labels_to_an_issue(repo_name, issue_number, labels)
+      end
+    end
+
+    def remove_labels_from_an_issue(repo_name, issue_name, labels)
+      Array(labels).each do |label|
+        if dry_run
+          puts "** dry-run: github.remove_label(#{repo_name.inspect}, #{issue_number.inspect}, #{label.inspect})".light_black
+        else
+          client.remove_label(repo_name, issue_number, label)
```
Comment (jrafanie) [inline lib/multi_repo/service/github.rb:155]: Should `issue_number` in the above 2 lines be `issue_name` ?

Comment (jrafanie) [reply lib/multi_repo/service/github.rb:155]: or maybe the argument should be called `issue_number` like in the add labels definition.

Comment (Fryguy) [reply lib/multi_repo/service/github.rb:155]: Oh yeah... How the heck did this work haha

**Comment by kbrock** [comment]
tangent: was curious how the code would work if there were a `Git` and a `DryRunGit` (child with commented out methods

File: lib/multi_repo/service/github.rb (hunk 1 above)
Comment (Fryguy) [reply lib/multi_repo/service/github.rb:155]: Done.

**Comment by Fryguy** [comment]
@kbrock Interesting idea - might be worth exploring in an issue.  I've always struggled with how best to architect "modifying" an existing gem's interface.  We did it 3 different ways in 3 different projects, and I never really liked any of them.  This one is my "favorite" so far, but still not great.

**Comment by kbrock** [comment]
ooh. I was referring to:

```ruby
class Git
  def destroy_remote(remote)
      if dry_run
        puts "** dry-run: git remote rm #{remote}".light_black
      else
        client.remote("rm", remote)
      end
    rescue MiniGit::GitError
      # Ignore missing remotes because we want them destroyed anyway
      nil
    end
  end
end

### becomes

def git
  @git ||= dry_run ? DryGit.new : Git.new
end

class Git
  def destroy_remote(remote)
    client.remote("rm", remote)
  rescue MiniGit::GitError
    # Ignore missing remotes because we want them destroyed anyway
    nil
  end
end

class DryGit
  def destroy_remote(remote)
    puts "** dry-run: git remote rm #{remote}".light_black
  end
end
```

But this may separate the dry from the actual run by too much so they may differ.
May need a spec to ensure they match up. `Git.methods(false).each { |method_name| expect(dryGit.new.respond_to?(method_name)).to be_truthy }`

**Pull request merged**